.circleci/config.yml

```diff
@@ -21,3 +21,6 @@ jobs:
         - run:
             name: CLI Tests
             command: make ci && pipenv run make integration
+        - run:
+            name: Unit Tests
+            command: make unit
```

Comment:
Is this needed? I think `make ci` will run the unit tests. I think the it will also run the integration tests, so I'm not sure why the previous command is running those again either 🤔

Reply:
Aha, you are probably right, I'll check it thanks. This is not ready for review yet, so don't waste your time, I am still trying out a few things, mostly to align pipenv - requirements.txt dependencies and workflow.